userguide/advanceduse/ssh.rst

```diff
@@ -70,6 +70,23 @@ In this case you use the second IP address
 
 Of course you can now also use ``scp`` or ``sshfs`` to transfer files.
 
+Configuration
+-------------
+
+you can enable ssh on boot by running the following command::
+    
+    sudo systemctl enable ssh.socket
+
+you can edit the port ssh is listening on with the following command::
+    
+    sudo systemctl edit ssh.socket
+
+then add the following text::
+
+    [Socket]
+    ListenStream=
```

Comment:
What does this line do? Is it needed?

Reply:
yes this line reset the socker it is listening one, without this line it would listen on both 22 and whatever the user sets